modules/exploits/linux/ssh/ubiquiti_airos_file_upload.rb

```diff
@@ -148,11 +148,10 @@ def ssh_login
         refs: self.references,
         info: ssh.transport.server_version.version
       )
-      report_note(
-        host: rhost,
-        port: datastore['SSH_PORT'],
-        type: 'airos.ssh.key',
-        data: private_key
+      store_valid_credential(
```

Comment:
In this case the module uses 2 library mixins, we don't yet provide helper service_details method on ssh and that looks to be the service you want this credential on.  May be worth adding an override for service_details port and protocol value here.

```
def service_details
  super.merge({ 
    service_name: 'ssh',
    port: datastore['SSH_PORT']
  })
end
```